betting/src/deploy/.env.dist

```diff
@@ -0,0 +1,11 @@
+# TZ_RPC=https://jakartanet.tezos.marigold.dev/
+TZ_RPC="http://127.0.0.1:5432"
```

Comment:
This adds cognitive load, as a person not familiar with tezos I don't know what should I run on this url / port
why not using default flextesa config as in https://github.com/ligolang/dao-cameligo/blob/main/.env.dist

Reply:
Fixed, new port is standard 20000

Reply:
I think we should repeat the comment that gives the flextesa url as an hint